src/robot/libraries/Remote.py

```diff
@@ -197,7 +197,10 @@ def _convert(self, value):
 class XmlRpcRemoteClient(object):
 
     def __init__(self, uri, timeout=None):
-        transport = TimeoutTransport(timeout=timeout)
+        if 'https' in uri:
```

Comment:
It would be safer to use `if uri.startswith('https://'):` here. Could even use `if urlparse(uri).scheme == 'https':` but that's perhaps an overkill and importing `urlparse` is a bit annoying because it is different modules depending on Python versions.